src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVM.cpp

```diff
@@ -59,7 +59,9 @@ template< typename BASE >
 SinglePhaseFVM< BASE >::SinglePhaseFVM( const string & name,
                                         Group * const parent ):
   BASE( name, parent )
-{}
+{
+  SinglePhaseBase::addLogLevel< logInfo::ResidualValues >();
```

Comment:
Why not using the `logInfo::ResidualValues` in this class?

Reply:
Why did you remove the `SinglePhaseBase::addLogLevel< logInfo::ResidualValues >();` instead of using it for this file logs?